app/uploaders/lighthouse_document_uploader.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+# [wipn8923] replaces app/uploaders/evss_claim_document_uploader.rb ?
```

Comment:
Confirmed, yes this is the equivalent; it's referenced in their [DocumentUpload](https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/workers/lighthouse/document_upload.rb) worker, which I ended up using in my branch.

Noting that these "uploaders" are confusingly named since as you can see in the worker linked above, we also use them to _retrieve_ attachments we originally stored in S3 when the veteran uploaded them. So I believe they are "uploaders" in the sense they wrap the CarrierWave functionality to upload S3 but on this end of the process, when we submit to EVSS/Lighthouse, we use them to retrieve the file from S3.